src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -249,7 +249,7 @@ protected function convertValidationExceptionToResponse(ValidationException $e,
     protected function invalid($request, ValidationException $exception)
     {
         return redirect($exception->redirectTo ?? url()->previous())
-                    ->withInput($request->except($this->dontFlash))
+                    ->withInput(array_except($request->input(), $this->dontFlash))
```

Comment:
the helper will be remove in `5.9`, might as well just use `Arr::except()`

Reply:
Done on 5.8 branch.